analysis/plotting.py

```diff
@@ -61,14 +61,21 @@ def _formatted_title(benchmark_snapshot_df):
 
 class Plotter:
     """Plotter that uses the same color for the same fuzzer."""
+    # Tableau 20 colors.
+    COLOR_PALETTE = [
```

Comment:
Consider using the "private" naming convention: _COLOR_PALETTE

Reply:
Done!